README.md

```diff
@@ -1 +1,103 @@
-# building2parcel-pix2pix
\ No newline at end of file
+# building2parcel-pix2pix
+
+This repository contains the implementation of CycleGAN and Pix2Pix models for image-to-image translation tasks. It provides comprehensive setup guides for both Windows and Linux environments.
+
+## Project Overview
+
+This project focuses on implementing and training CycleGAN and Pix2Pix models for various image-to-image translation tasks. These models can be used for a wide range of applications, including style transfer, object transfiguration, season transfer, and photo enhancement.
+
+## Environment Setup
+
+Choose the appropriate guide based on your operating system:
+
+- [Windows Setup Guide](./docs/Windows_README.md)
+- [Linux Setup Guide](./docs/Linux_README.md)
+
+## Quick Start
+
+1. Clone this repository:
+
+   ```
+   git clone https://github.com/scalable-design-participation-lab/re-blocking.git
```

Comment:
doesn't this need to be this repo instead??? `https://github.com/scalable-design-participation-lab/building2parcel-pix2pix`